openwisp_firmware_upgrader/migrations/0003_default_permissions.py

```diff
@@ -0,0 +1,52 @@
+from django.db import migrations
+from django.contrib.auth.models import Permission
+
+from . import create_default_permissions
+
+
+
+def assign_permissions_to_groups(apps, schema_editor):
+    create_default_permissions(apps, schema_editor)
+    Group = apps.get_model('openwisp_users', 'Group')
+
+    try:
+        admin = Group.objects.get(name='Administrator')
+        operator = Group.objects.get(name='Operator')
+    # consider failures custom cases
+    # that do not have to be dealt with
+    except Group.DoesNotExist:
+        return
+
+    operators_read_only_admins_manage = ['build', 'devicefirmware', 'firmwareimage', 'batchupgradeoperation', 'upgradeoperation']
+    admins_can_manage = ['category']
+    manage_operations = ['add', 'change', 'delete']
+
+    for action in manage_operations:
+        for model_name in admins_can_manage:
+            permission = Permission.objects.get(
+                codename='{}_{}'.format(action, model_name)
```

Comment:
this is missing:

```
content_type__app_label=app_label
```

Please add it here and to all the other `Permission.objects.get` calls